cypress/integration/pipeline.triggers.spec.ts

```diff
@@ -22,7 +22,7 @@ import {
 let headers = {};
 const TRIGGER_PIPELINE_1 = `trigger_test_pipeline_${Date.now()}`;
 const TRIGGER_PIPELINE_2 = `trigger_test_pipeline_2_${Date.now()}`;
-describe('Pipeline Studio', () => {
+describe.skip('Pipeline Studio', () => {
```

Comment:
This will no longer pass after setting `feature.pipeline.composite.triggers.enabled` to `true` in `cdap.json`. It uses the old triggers panel.